sdcm/tester.py

```diff
@@ -1605,14 +1606,17 @@ def run_stress_thread(self, stress_cmd, duration=None, stress_num=1, keyspace_nu
     # pylint: disable=too-many-arguments
     def run_stress_cassandra_thread(
             self, stress_cmd, duration=None, stress_num=1, keyspace_num=1, profile=None, prefix='', round_robin=False,
-            stats_aggregate_cmds=True, keyspace_name=None, stop_test_on_failure=True, **_):
+            stats_aggregate_cmds=True, keyspace_name=None, stop_test_on_failure=True, loaders_tenant=0, **_):
         # stress_cmd = self._cs_add_node_flag(stress_cmd)
         timeout = self.get_duration(duration)
         if self.create_stats:
             self.update_stress_cmd_details(stress_cmd, prefix, stresser="cassandra-stress",
                                            aggregate=stats_aggregate_cmds)
         stop_test_on_failure = False if not self.params.get("stop_test_on_stress_failure") else stop_test_on_failure
-        cs_thread = CassandraStressThread(loader_set=self.loaders,
+        self.log.info("Loaders tenant index: %s", loaders_tenant)
+        self.log.info("Loaders set: %s", self.loaders_multitenant[loaders_tenant])
+        loader_set = self.loaders_multitenant[loaders_tenant] if self.loaders_multitenant else self.loaders
+        cs_thread = CassandraStressThread(loader_set=loader_set,
```

Comment:
handling the `loader_set` isn't enough.

see below, it's targeting only the first node:
```
node_list=self.db_cluster.nodes
```